scpd/lms/static/sass/partials/base/_variables.scss

```diff
@@ -1,9 +1,10 @@
-// Fonts
-$sans-serif: 'Source Sans Pro', Trebuchet, 'Helvetica Neue', Helvetica, $verdana, sans-serif;
-$f-sans-serif: $sans-serif;
-$serif: 'Crimson Text', Georgia, Cambria, 'Times New Roman', Times, serif;
-$f-serif: $serif;
-$header-sans-serif: $sans-serif;
-
-// Import defaults
+@import 'variable_overrides/palette.scss';
+
+@import 'variable_overrides/colors.scss';
+
```

Comment:
Nit: but do we need newlines between every one of these imports?
eg. in a Python file, we would group related imports.